src/main/kotlin/com/klaxit/hiddensecrets/HiddenSecretsPlugin.kt

```diff
@@ -24,22 +25,28 @@ open class HiddenSecretsPlugin : Plugin<Project> {
         private const val KOTLIN_FILE_NAME = "Secrets.kt"
 
         //Tasks
+        const val TASK_GROUP = "Hide secrets"
         const val TASK_UNZIP_HIDDEN_SECRETS = "unzipHiddenSecrets"
         const val TASK_COPY_CPP = "copyCpp"
         const val TASK_COPY_KOTLIN = "copyKotlin"
         const val TASK_HIDE_SECRET = "hideSecret"
+        const val TASK_HIDE_SECRET_FROM_PROPERTIES_FILE = "hideSecretFromPropertiesFile"
         const val TASK_OBFUSCATE = "obfuscate"
         const val TASK_PACKAGE_NAME = "packageName"
         const val TASK_FIND_KOTLIN_FILE = "findKotlinFile"
 
         //Properties
-        private const val KEY = "key"
-        private const val KEY_NAME = "keyName"
-        private const val PACKAGE = "package"
+        private const val PROP_KEY = "key"
+        private const val PROP_KEY_NAME = "keyName"
+        private const val PROP_PACKAGE = "package"
+        private const val PROP_FILE_NAME = "propertiesFileName"
```

Comment:
_**"propertiesFileName"**_ ne suit pas les autres, c'est plutot **_"fileName"_** non ?

Reply:
@GhassenMsd it is a property file it can stay like that to be as clear as possible.